router/router.go

```diff
@@ -0,0 +1,29 @@
+package router
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/rayl17/academy-go-q42021/controller"
+	parameters "github.com/rayl17/academy-go-q42021/global"
+)
+
+func NewRouter(c controller.ControllerInterface) *http.Server {
+
+	router := mux.NewRouter()
+
+	router.HandleFunc("/pokemon/{name}", c.GetPokemonByName).Methods("GET")
+	router.HandleFunc("/pokemonid/{id}", c.GetPokemonByID).Methods("GET")
```

Comment:
pokemonid is not an entity (in this project).

```suggestion
	router.HandleFunc("/pokemon/name/{name}", c.GetPokemonByName).Methods("GET")
	router.HandleFunc("/pokemon/id/{id}", c.GetPokemonByID).Methods("GET")
```